app/services/org/create_created_plan_service.rb

```diff
@@ -6,24 +6,54 @@ def call(org = nil)
 
         orgs.each do |org|
           OrgDateRangeable.split_months_from_creation(org) do |start_date, end_date|
-            create_count_for_date(start_date: start_date, end_date: end_date, org: org)  
+            create_or_update(start_date: start_date, end_date: end_date, org: org)
           end
         end
       end
 
       private
 
+      def users(org)
+        User.where('users.org_id = ?', org.id)
+      end
+
+      def plans(start_date:, end_date: )
+        Plan.where('plans.created_at >= ? AND plans.created_at <= ?', start_date, end_date)
+      end
+
+      def creator_admon
```

Comment:
Think this should be called `creator_admin` :)

Reply:
I guess so